apps/gateway/src/chat/tools/parse-provider-response.ts

```
@@ -3,6 +3,7 @@ import { logger } from "@llmgateway/logger";
 
 import { estimateTokens } from "./estimate-tokens.js";
 import { adjustGoogleCandidateTokens } from "./extract-token-usage.js";
+import { extractMinimaxThinking } from "./extract-reasoning.js";
```

ESLint

[error] `./extract-reasoning.js` import should occur before import of `./extract-token-usage.js`

(import/order)